From 66df315d2a270a254c613a4d2e72c0ea47f15a71 Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Thu, 27 Mar 2014 09:35:29 +0000
Subject: [PATCH] vtysh/vtysh.c: works with new readline

The Function and CPPFunction had been removed by in readline 6.3, use
the new functions to replace them.

Upstream-Status: Pending

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 vtysh/vtysh.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/vtysh/vtysh.c b/vtysh/vtysh.c
index 431c08e..fdd82fb 100644
--- a/vtysh/vtysh.c
+++ b/vtysh/vtysh.c
@@ -2212,9 +2212,9 @@ void
 vtysh_readline_init (void)
 {
   /* readline related settings. */
-  rl_bind_key ('?', (Function *) vtysh_rl_describe);
+  rl_bind_key ('?', (rl_command_func_t *) vtysh_rl_describe);
   rl_completion_entry_function = vtysh_completion_entry_function;
-  rl_attempted_completion_function = (CPPFunction *)new_completion;
+  rl_attempted_completion_function = (rl_completion_func_t *)new_completion;
   /* do not append space after completion. It will be appended
    * in new_completion() function explicitly. */
   rl_completion_append_character = '\0';
-- 
1.8.3.4

